lib/matplotlib/legend.py

```diff
@@ -867,10 +876,36 @@ def get_texts(self):
         r"""Return the list of `~.text.Text`\s in the legend."""
         return silent_list('Text', self.texts)
 
+    def set_alignment(self, alignment):
+        """
+        Set the alignment of legend box. Elements will align
+        to the longest one.
+
+        Parameters
+        ----------
+        alignment : str
+            One of {'center', 'left', 'right'}.
```

Comment:
```suggestion
        alignment : {'center', 'left', 'right'}.

```
It's ok to have no parameter description if there is really nothing left to say.